sdk/@launchdarkly/observability-dotnet/src/LaunchDarkly.Observability/ObservabilityExtensions.cs

```diff
@@ -112,7 +112,8 @@ internal static void AddLaunchDarklyObservabilityWithConfig(this IServiceCollect
                     .AddWcfInstrumentation()
                     .AddQuartzInstrumentation()
                     .AddAspNetCoreInstrumentation(options => { options.RecordException = true; })
-                    .AddSqlClientInstrumentation(options => { options.SetDbStatementForText = true; });
+                    .AddSqlClientInstrumentation(options => { options.SetDbStatementForText = true; })
+                    .AddSource(DefaultNames.ActivitySourceNameOrDefault(config.ServiceName));
```

Comment:
This line accidentally got removed in merge conflict resolution. So re-adding it.